x-pack/plugins/observability/server/index.ts

```diff
@@ -48,8 +48,8 @@ const configSchema = schema.object({
     }),
     thresholdRule: schema.object({
       enabled: offeringBasedSchema({
-        serverless: schema.boolean({ defaultValue: true }),
-        traditional: schema.boolean({ defaultValue: true }),
+        serverless: schema.boolean({ defaultValue: false }),
```

Comment:
Just out of curiosity, why is the default false?

Reply:
I wanted to make sure it works without using feature flag. 😄